From 779f4516fc947e3134febae0a78ed3f6ae7e0fab Mon Sep 17 00:00:00 2001
From: Hongcheng Xie <hongcheng.xie@intel.com>
Date: Wed, 9 Sep 2020 11:19:23 +0800
Subject: [PATCH] Temp don't call tetherGetStats API

Temp don't call tetherGetStats API in NetdService to avoid
iptable terminated problem in netd.

Change-Id: I23a98feff4027ae0eed218eb890abb2c55b07921
Tracked-On: ACP-1275
Signed-off-by: Hongcheng Xie <hongcheng.xie@intel.com>
---
 .../core/java/com/android/server/NetworkManagementService.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/NetworkManagementService.java b/services/core/java/com/android/server/NetworkManagementService.java
index 0ddfa1c16a0a..dc3049014d8e 100644
--- a/services/core/java/com/android/server/NetworkManagementService.java
+++ b/services/core/java/com/android/server/NetworkManagementService.java
@@ -1491,7 +1491,7 @@ public class NetworkManagementService extends INetworkManagementService.Stub {
         public NetworkStats getTetherStats(int how) {
             // We only need to return per-UID stats. Per-device stats are already counted by
             // interface counters.
-            if (how != STATS_PER_UID) {
+            if (true/*temp WA*/ || how != STATS_PER_UID) {
                 return new NetworkStats(SystemClock.elapsedRealtime(), 0);
             }
 
-- 
2.17.1

